### ISSUE TITLE
[ci] enable API tests for windows and mac

### ISSUE THREAD

**Issue by akosyakov** [eclipse-theia/theia#7646]

**Comment by DoroNahari** [comment]
Tried to run on mac. one test failed:
![image](https://user-images.githubusercontent.com/24614792/80300112-80963200-87a2-11ea-9c4b-f72dfaa850c4.png)
@amiramw

**Comment by akosyakov** [comment]
@DoroNahari yes we will need to adjust keybinding expectations according to OS
